=== modified file 'lib/libbamf/bamf-application.c'
--- lib/libbamf/bamf-application.c	2013-06-07 07:36:18 +0000
+++ lib/libbamf/bamf-application.c	2013-06-08 15:40:18 +0000
@@ -641,6 +641,11 @@
 
   name = g_strdup (g_app_info_get_name (G_APP_INFO (desktop_info)));
 
+/* GLib doesn't provide this by default */
+#ifndef G_KEY_FILE_DESKTOP_KEY_FULLNAME
+#define G_KEY_FILE_DESKTOP_KEY_FULLNAME "X-GNOME-FullName"
+#endif
+
   /* Grab the better name if its available */
   fullname = g_key_file_get_locale_string (keyfile, G_KEY_FILE_DESKTOP_GROUP,
                                            G_KEY_FILE_DESKTOP_KEY_FULLNAME, NULL, NULL);

